05/DB.test.js

```diff
@@ -1 +1,75 @@
-import DB from './DB';
\ No newline at end of file
+import DB from "./DB";
+
+it("check if id is a number", async () => {
+  expect.assertions(1);
+  const db = new DB();
+  const id = "string";
+
+  try {
+    await db.insert({ id: id });
+  } catch (err) {
+    expect(err).toBe("ID can be only number!");
+  }
+});
+
+it("check if id is not duplicated", async () => {
+  const db = new DB();
+  const rowsLength = db.getRows().length;
+  expect.assertions(rowsLength);
+  const id = 3;
+
+  try {
+    await db.insert({ id: id });
+  } catch (err) {
+    expect(err).toBe("ID can't be duplicated!");
+  }
+});
+
+it("check if id found", async () => {
+  expect.assertions(1);
+  const db = new DB();
+  const id = 3;
+
+  try {
+    await db.select(id);
+  } catch (err) {
+    expect(err).toBe("ID not found");
+  }
+});
+
+it("check if item exist", async () => {
+  const db = new DB();
+  const rowsLength = db.getRows().length;
+  expect.assertions(rowsLength);

```

Comment:
Na tym etapie `rowsLength` to `undefined` więc nie działa to jak należy bo mamy `expect.assertions(undefined);` - tak nie powinniśmy pisać. To co poniżej jest ok.